README.md

```diff
@@ -4,12 +4,12 @@
 <img alt="Build Status" src="https://prow.k8s.io/badge.svg?jobs=ci-kind-build">
 </a> <a href="https://godoc.org/sigs.k8s.io/kind"><img src="https://godoc.org/sigs.k8s.io/kind?status.svg"></a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a></p>
 
-### [View The Documentation](https://kind.sigs.k8s.io)
+# Please see [Our Documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) for more in-depth installation etc.
 
 kind is a tool for running local Kubernetes clusters using Docker container "nodes".  
 kind is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
 
-If you have [go] and [docker] installed `GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0 && kind create cluster` is all you need!
+If you have [go] ([1.11+][go-supported]) and [docker] installed `GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0 && kind create cluster` is all you need!
```

Comment:
this is _another_ place we have to keep updated, I'm tempted to remove this shoutout entirely in the future once we have the podman etc support landed in favor of just linking to the install guide in the quick start...

for now we should keep this clear though, and we're pointing out _why_ (we follow the upstream support policy for now of the 2 latest go minor versions, in the future since we have the containerized build it might be reasonable to just focus on a specific version and tell people to use that...)